contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -63,9 +63,9 @@ contract FunctionsRouter is IFunctionsRouter, FunctionsSubscriptions, Pausable,
   error GasLimitTooBig(uint32 limit);
 
   struct CallbackResult {
-    bool success;
-    uint256 gasUsed;
-    bytes returnData;
+    bool success; // ══════╸ Whether the callback succeeded or not
```

Comment:
Nit: why the switch to double lines and lines for items that are not packed with other items? 

Double lines are more of a stylistic choice (and I don't care much about it) but I Personally find the lines for things that are unpacked fairly ugly. Won't be blocking the PR of course, all just decoration

Reply:
While maybe uglier, I find it a lot more clear. When there are no lines, at a glance it looks like the struct could have not been packed efficiently.

Reply:
Think of it as denoting the slot, rather than what is packed together